### DISCUSSION TITLE
error 404

### DISCUSSION THREAD

**Discussion by angelessiena** [community/community#68349] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

quiero subir mi repo a github, pero al ingresar al link de la pág. que me genera github, me muestra el index, pero luego el resto de los enlaces no puedo acceder porque me aparece error 404, que puedo hacer? ya miré las recomendaciones que dá github en la página pero ya probé todo. Borré cache, el nombre de index está bien (todo minúsculas), no se cual puede ser el error.

**Comment by radar-blip** [comment]
Try changing this link:
`<a class="nav-link-inicio" aria-current="page" href="../clase-7/pages/inicio.html">Inicio</a>`

to this:
`<a class="nav-link-inicio" aria-current="page" href="pages/inicio.html">Inicio</a>`

do the same to the rest of them.

**Reply by angelessiena** [reply]
si lo tengo cambiado en visual y aún sigue con errores :(

**Reply by angelessiena** [reply]
ahora en visual studio code me aparece que tengo 6 archivos sin guardar, cuando inserto el git add . para guardar o adherir, me dice: fatal, no es un repositorio git....
no se la verdad donde está el problema..

**Reply by radar-blip** [reply]
In your index.html your links are pointing to a clase-7 folder, there is no clase-7 folder in your repository so that is where the error is occurring.

Try changing this link:
`<a class="nav-link-inicio" aria-current="page" href="../clase-7/pages/inicio.html">Inicio</a>`

to this:
`<a class="nav-link-inicio" aria-current="page" href="pages/inicio.html">Inicio</a>`

do the same to the rest of them.
